katalog/harbor/jobservice/kustomization.yaml

```
@@ -9,27 +9,33 @@ kind: Kustomization
 resources:
   - deploy.yml
   - svc.yml
+  - pvc-logs.yml
+  - pvc-scandata.yml
 
 images:
   - name: goharbor/harbor-jobservice
     newName: registry.sighup.io/fury/goharbor/harbor-jobservice
-    newTag: v2.4.2
+    newTag: v2.7.0
 
 configMapGenerator:
   - name: jobservice
     files:
       - config.yml=config/config.yml
     literals:
       - CORE_URL=http://core
+      - REGISTRY_URL=http://registry:5000
+      - TOKEN_SERVICE_URL=http://core/service/token
       - REGISTRY_CONTROLLER_URL=http://registry:8080
       - LOG_LEVEL=info
+      - METRIC_NAMESPACE=harbor
+      - METRIC_SUBSYSTEM=jobservice
       - HTTP_PROXY=
       - HTTPS_PROXY=
-      - NO_PROXY=core,jobservice,database,chartmuseum,clair,notary-server,notary-signer,registry,portal,127.0.0.1,localhost,.local,.internal
+      - NO_PROXY=core,jobservice,database,chartmuseum,trivy,notary-server,notary-signer,registry,portal,exporter,127.0.0.1,localhost,.local,.internal
 
 secretGenerator:
   - name: jobservice
     literals:
       - secret=Gx6IsNtY4NdWoK0u
       - REGISTRY_CREDENTIAL_USERNAME=harbor_registry_user
-      - REGISTRY_CREDENTIAL_PASSWORD=DhQTxjwUAyUZHaMABvm90dGAJMDX5jaa
+      - REGISTRY_CREDENTIAL_PASSWORD=DhQTxjwUAyUZHaMABvm90dGAJMDX5jaa
```

missing new line at the end of the file
```suggestion
      - REGISTRY_CREDENTIAL_PASSWORD=DhQTxjwUAyUZHaMABvm90dGAJMDX5jaa

```